sambaplayersdk/src/main/java/com/sambatech/player/SambaSimplePlayerView.java

```diff
@@ -800,7 +802,27 @@ public void destroyCastPlayer(){
         sambaCastPlayer = null;
     }
 
-    public void showCast() {
+    public void showCast(boolean isLive, final CastLiveButtonListener castLiveButtonListener) {
+
+        if (isLive) {
+            castControlView.findViewById(R.id.exo_progress_controls).setVisibility(View.GONE);
+            View castButton = castControlView.findViewById(R.id.topbar_live_button);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "castButton" which hides the field declared at line 90. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AHiddenFieldCheck)